packages/data-transport-layer/src/services/main/service.ts

```diff
@@ -73,17 +77,23 @@ export class L1DataTransportService extends BaseService<L1DataTransportServiceOp
       }
     })
 
+    this.state.failureGauge = new this.state.metrics.client.Gauge({
+      name: 'data_transport_layer_main_service_failures',
+      help: 'Counts the number of times that the main service fails',
+      registers: [this.state.metrics.registry],
+    })
+
```

Comment:
is there a reason why this is a gauge instead of a counter?

Reply:
You are right, good call